src/components/SliderHero/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Container, Flex, Heading, Stack, Text } from '@chakra-ui/react'
+
+import { ChakraNextImage } from '@components'
+
+interface SliderHeroProps {
+  items: SubpageType[] | undefined
+  index: number
+}
+
+export const SliderHero = ({ items, index }: SliderHeroProps): JSX.Element => {
+  return (
+    <Container display={{ base: 'none', md: 'initial' }} maxW={'7xl'} mt={10}>
+      <Stack
+        align={'center'}
+        spacing={{ base: 8, md: 10 }}
+        direction={{ base: 'column', md: 'row' }}
+      >
+        <Stack flex={1} spacing={{ base: 5, md: 10 }}>
+          <Heading
+            lineHeight={1.1}
+            fontWeight={600}
+            fontSize={{ base: '3xl', sm: '4xl', lg: '6xl' }}
+          >
+            <Text as={'span'} color={'primary'}>
+              {items && items[index]?.title}
+            </Text>
+          </Heading>
+          <Text noOfLines={5} fontSize={'1.3rem'}>
+            {items && items[index]?.content}
```

Comment:
> **NOTE: We can discuss and solve it with another pull request. Actually this is not an issue I'm expecting you to take for this PR**

Here I'm worried about UX. Let's discuss about it:

- User may want to click any SliderItem card to see it as Hero. In this case we redirect user to the details page.
- If user wants to visit details of any slider item by clicking it, we don't indicate `Read More` button. 

**Possible solution may be:**

1. Remove `<Navigate>` which wraps Card component.
2. Put a `Read More` button underneath `SliderHero` content. (We will need link for SliderHero)
3. SliderItem will have Navigate if it doesn't receive `withHero` prop.

> Card component needs to be reconsidered in terms of navigating in this case.

Reply:
What do you think? Any other suggestions? @MustafaTRKYLMZ @ismkl @yuskara

Reply:
By the way. I've just noticed that we may also achieve similar result by using built in solution here: https://swiperjs.com/react#thumbs

Reply:
I think we should try some diffirent UX. Because It is so important. That is why, we should disscuss firstly and than If it is neccessary we can create new issue about it.